Makefile.source.mjs

```diff
@@ -389,6 +389,54 @@ target["test-cov"] = function () {
   );
 };
 
+function bootstrapParserTests(name, repoURL, subPaths) {
+  function getParserTestsCommit(id) {
+    const content = readFileSync("./Makefile", "utf8");
+    const commit = content.match(new RegExp(`${id}_COMMIT = (\\w{40})`))[1];
+    if (!commit) throw new Error(`Could not find ${id}_COMMIT in Makefile`);
+    return commit;
+  }
+
+  const dir = "./build/" + name.toLowerCase();
+
+  shell.rm("-rf", dir);
+  shell.mkdir("-p", "build");
+
+  exec("git", [
+    "clone",
+    "--filter=blob:none",
+    "--sparse",
+    "--single-branch",
+    "--shallow-since=2021-05-01",
```

Comment:
```suggestion
    "--shallow-since='2 years ago'",
```

If this ever fails, it means we have forgotten to update the test targets, which we should.